client/src/components/HistoricElementView/tests/HistoricElementView-test.js

```diff
@@ -1,54 +1,41 @@
-/* global jest, describe, beforeEach, it, expect */
+/* global jest, test, describe, beforeEach, it, expect */
 
 import React from 'react';
-import ReactTestUtils from 'react-dom/test-utils';
 import ElementalAreaHistoryFactory from '../HistoricElementView';
+import { render } from '@testing-library/react';
 
-describe('HistoricElementView', () => {
-  class FieldGroupStub extends React.Component {
-    getLegend() {
-      return 'nah';
-    }
-    getClassName() {
-      return 'ok';
-    }
-    render() {
-      return <div>Group</div>;
-    }
-  }
-  const datumSet = {
-    ElementType: 'Stub',
-    ElementTitle: 'Pretend Element',
-    ElementEditLink: 'http://localhost:8080/'
+function makeProps(obj = {}) {
+  return {
+    data: {
+      ElementID: 1,
+      ElementType: 'Stub',
+      ElementTitle: 'Pretend Element',
+      ElementEditLink: 'http://localhost:8080/',
+      tag: 'div'
```

Comment:
Do you recall why you added `ElementID` and `tag` which aren't in the original test?
No biggie if not, that's more me being curious than anything.